sn_client/src/wallet.rs

```diff
@@ -139,7 +140,10 @@ impl WalletClient {
     pub async fn pay_for_storage(
```

Comment:
I think we need to update the documentation here. It's not immediately obvious what more we're returning here. (original docs were probs insuffcient here too though)